### PR TITLE
[frr] FRR plugin

### PR DESCRIPTION
**Pull request by ins-pirat-ion**
Fedora is going to ship FRR alternative to Quagga. Path to conf
is /etc/frr/.

Signed-off-by: Filip Krska <fkrska@redhat.com>

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?

### CODE REVIEW

**Comment by ins-pirat-ion** [comment]
Can be tested with

https://copr.fedorainfracloud.org/coprs/mruprich/frr/

**Comment by pmoravec** [comment]
+1

File: sos/plugins/frr.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2007 Ranjith Rajaram <rrajaram@redhat.com>
```
Comment (TurboTurtle) [inline sos/plugins/frr.py]: Minor nit, copyright line needs to be updated/corrected.

I guess this is a c'n'p error?

Comment (ins-pirat-ion) [reply sos/plugins/frr.py]: @TurboTurtle thanks for heads-up. I've skipped the personal (c) lines now (e.g. like it is in acpid.py). I'd rather not claim copy monopoly on s/quagga/frr/g job ;). And hopefully it's better for eventual future c'n'p jobs ;).

**Pull request closed without merging**